packages/maps/src/Client.ts

```diff
@@ -92,3 +92,56 @@ export interface MapOptions {
     geometryType: boolean
   };
 }
+
+export interface MapInstance {
```

Comment:
Really nice addition. I thought about integrating it at the beginning, but I didn't.